res/gettext/fi.po

```diff
@@ -7,270 +7,131 @@ msgid ""
 msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-11-26 08:21+0200\n"
-"PO-Revision-Date: 2020-11-26 08:21+0200\n"
+"POT-Creation-Date: 2020-11-28 16:42+0200\n"
+"PO-Revision-Date: 2020-11-28 16:43+0200\n"
 "Last-Translator: \n"
 "Language-Team: \n"
 "Language: fi\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 2.3\n"
+"X-Generator: Poedit 2.0.6\n"
```

Comment:
Jösses mikä diffi yhden käännöksen lisäämisestä. Johtuuko tää tästä Poedit-versiomuutoksesta? Ja aikooko ne jatkossakin aina randomoitua joka kerta, kun käännöksiä muokkaa eriversioisella editorilla kuin aiemmin? Pitäiskö ehkä editoida näitä .po-filuja käsin eikä käännöseditorilla, ettei näin kävisi?

Olisi siis jättekiva, jos diffistä näkisi selkeämmin, mitä tapahtuu. Mun puolesta ei tartte siivota tätä committia, mut jatkoa ajatellen vähän kuumottelee tää.